lib/Driver/ToolChains.cpp

```diff
@@ -1471,6 +1530,266 @@ toolchains::Darwin::constructInvocation(const LinkJobAction &job,
   return II;
 }
 
+ToolChain::InvocationInfo
+toolchains::Windows::constructInvocation(const InterpretJobAction &job,
+                                             const JobContext &context) const {
+  InvocationInfo II = ToolChain::constructInvocation(job, context);
+
+  SmallString<128> runtimeLibraryPath;
+  getRuntimeLibraryPath(runtimeLibraryPath, context.Args, *this);
+
+  addPathEnvironmentVariableIfNeeded(II.ExtraEnvironment, "PATH",
+                                     ":", options::OPT_L, context.Args,
+                                     runtimeLibraryPath);                                            
```

Comment:
Please don't do this, this way (via the environment).  We use use `-libpath` or `-L`.

Reply:
That doesn't work for the interpreter.

Reply:
Fixing this is probably out of scope for the PR as I'm only able to test cross-compiling at the moment, so testing the interpreter isn't an option.